### PR TITLE
chore: add batch instructions to stats

### PR DESCRIPTION
**Pull request by ze97286**

### CODE REVIEW

File: protos/sources/vega/api/v1/core.proto [hunk 1]
```diff
@@ -324,6 +324,10 @@ message Statistics {
   string epoch_start_time = 37;
   // Epoch expected end time
   string epoch_expiry_time = 38;
+  // Number of events in the last block
+  uint64 event_count = 39;
+  // Rate of events per second in total
```
Comment (candida-d) [inline protos/sources/vega/api/v1/core.proto:329]: Don't quite understand this description, could you perhaps reword, or explain in a comment and I can provide a suggestion?

**Review by jeremyletang — DISMISSED** [review]

File: protos/sources/vega/api/v1/core.proto (hunk 1 above)
Comment (ze97286) [reply protos/sources/vega/api/v1/core.proto:329]: `number_of_events_emitted_in_last_block / duration_in_seconds_last_block`

Comment (candida-d) [reply protos/sources/vega/api/v1/core.proto]: Thank you!

**Review by candida-d — APPROVED** [review]

**Review by EVODelavega — APPROVED** [review]

**Comment by edd** [comment]
I've added these to the stats page with the current logic:
- events/second & eventCount shown as ✅  if the event count is less than (`txsPerSecond` * `59`), red if above

Magic number 59 was got from two numbers @ze97286 gave me as an example, so they might not be good. It can be changed here: https://github.com/vegaprotocol/stats/blob/f8eca96c4ffab3a97a44e0c702ccaaaec382a529/src/index.html#L346-L347

**Pull request merged**